Userland/Libraries/LibJS/Runtime/RegExpPrototype.cpp

```diff
@@ -1075,6 +1099,23 @@ JS_DEFINE_NATIVE_FUNCTION(RegExpPrototype::compile)
         if (!flags.is_undefined())
             return vm.throw_completion<TypeError>(ErrorType::NotUndefined, flags.to_string_without_side_effects());
 
+        // https://github.com/tc39/proposal-regexp-legacy-features#regexpprototypecompile--pattern-flags-
+        // 3. Let thisRealm be the current Realm Record.
```

Comment:
Why is this inside the branch? Should be outside after step 2 above, current step 3 becomes step 7 (please update the numbers)